tests/assets/neuron/config.yaml

```diff
@@ -0,0 +1,65 @@
+{{$uniformQps := DefaultParam .CL2_UNIFORM_QPS 500}}
+{{$neuronResourcesPerPod := DefaultParam .CL2_NEURON_RESOURCES_PER_POD 64}}
+{{$neuronPods := DefaultParam .CL2_NEURON_PODS .Nodes}}
+
+name: neuron-workers
+namespace:
+  number: 1
+tuningSets:
+- name: UniformQPS
+  qpsLoad:
+    qps: {{$uniformQps}}
+
+steps:
+- name: Start measurements
+  measurements:
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = neuron-worker
+      threshold: 60s
```

Comment:
Upstream SLOs is <=5sec.
Wondering why are we doing `60sec` ?

Reply:
`25seconds` is still not under upstream SLO. W should aim for upstream SLO unless we have a strong reason/doc saying otherwise why its not possible.